stats/kernel-selftests

```diff
@@ -1101,6 +1092,20 @@ class KmodStater < Stater
       super(line, stats)
     end
   end
+
+  def running_multiple_test(test_case, stats)
+    @last_test_case = @test_case
```

Comment:
thanks, by looking at the new code, i think it's no need to use instance variable for @last_test_case. Here can just use last_test_case.